pkg/ui/react-app/src/thanos/pages/errorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ErrorInfo } from 'react';
+import { Container, UncontrolledCollapse, Button } from 'reactstrap';
+import styles from './ErrorBoundary.module.css';
+
+interface ErrorState {
+  error: Error | null;
+  errorInfo: ErrorInfo | null;
+}
+
+class ErrorBoundary extends React.Component<{}, ErrorState> {
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      error: null,
+      errorInfo: null,
+    };
+  }
+
+  componentDidCatch(error: Error, errorInfo: ErrorInfo): void {
+    this.setState({
+      error,
+      errorInfo,
+    });
+  }
+  render(): React.ReactNode {
+    if (this.state.errorInfo) {
+      return (
+        <Container fluid className={styles.container}>
+          <h1>Aaaah! Something went wrong.</h1>
+          <h3>
+            Please file an issue at&nbsp;
+            <a href="https://github.com/thanos-io/thanos/issues" target="_blank" rel="noreferrer noopener">
+              Thanos issue tracker.
+            </a>
+          </h3>
+          <Button color="link" id="toggler" className={styles.detailsBtn} style={{ fontSize: '1.2em' }}>
```

Comment:
If this is the only instance of the detailsBtn class, why split the font size out of the css? To keep the css class more generic?

Reply:
The internal bootstrap classes were getting priority over the `detailsBtn` class, so I wasn't able to modify the font-size here. I had two options, either use `!important` on use inline CSS.

I think maybe there is a third solution too. I can increase the specificity of `detailsBtn` class, by declaring it as `button.detailsBtn` or something.

Reply:
Using `button.detailsBtn` instead of just `.detailsBtn` worked :tada:

Reply:
yes let's avoid !important for sure. making it more specialized :+1: